filter/src/main/java/com/fightpandemics/filter/domain/LocationPredictionsUseCase.kt

```diff
@@ -24,7 +25,11 @@ class LocationPredictionsUseCase @Inject constructor(
         return locationRepository.getLocationPredictions(parameters!!)!!.map {
             when (it) {
                 is Result.Loading -> it
-                is Result.Success -> it
+                is Result.Success -> {
+                    val predictions = it.data as List<Prediction>
```

Comment:
Do I understand this correctly that first the data of the result is a list of `Prediction`s which is turned into a list of their descriptions? Could this be done earlier or does it need to happen here? Maybe would also be nice to make into it's own function such as `getDescriptionsFromPredictions` or similar.